core/src/com/unciv/logic/trade/TradeEvaluation.kt

```diff
@@ -273,7 +286,19 @@ class TradeEvaluation {
                     ThreatLevel.Low -> 250
                     ThreatLevel.Medium -> 500
                     ThreatLevel.High -> 1000
-                    ThreatLevel.VeryHigh -> 10000 // no way boyo
+                    ThreatLevel.VeryHigh -> 10000 // declaring war to a super power is NO WAY
+                }
+            }
+
+            TradeType.PeaceDeclaration -> {
+                val civToMakePeaceWith = civInfo.gameInfo.getCivilization(offer.name)
+
+                return when (Automation.threatAssessment(civInfo, civToMakePeaceWith)) {
```

Comment:
This looks like a way to exploit the AI, honestly. Would you agree to this? 
We're trying to make an AI that plays to win, so it should be dependent on e.g
 how close I am to taking over a city